package.json

```diff
@@ -22,12 +15,12 @@
     "build": "yarn run build-clean && yarn build-ts && yarn build-js",
     "build-js": "lerna run build --scope codemirror-graphql --scope graphql-language-service-server --scope graphiql graphql-language-service",
     "build-ts": "tsc --build",
-    "build-clean": "tsc --build --clean && rm -rf 'packages/**/src/{dist,esm,bundle}' && lerna run build-clean --parallel",
+    "build-clean": "tsc --build --clean && rimraf 'packages/**/{dist,esm,bundle,coverage}' && lerna run build-clean --parallel",
```

Comment:
actual clean build. wow not sure where that slipup happened but that's an important one